chef/cookbooks/neutron/recipes/common_config.rb

```diff
@@ -185,56 +180,61 @@
 num_vlans = neutron[:neutron][:num_vlans]
 vlan_end = [vlan_start + num_vlans - 1, 4094].min
 
+
 case neutron[:neutron][:networking_plugin]
```

Comment:
I would like to fixed the cleanup things when we are done with this PR. The change is already huge...